glog-config.cmake.in

```diff
@@ -8,4 +8,7 @@ include (CMakeFindDependencyMacro)
 
 @gflags_DEPENDENCY@
 
+list (APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}")
+find_dependency (Libunwind)
+
```

Comment:
Do you assume that the consumers of glog must create a copy of `FindLibunwind.cmake`? If so, this is not very user-friendly. The find module should be installed alongside glog and found automatically.

Reply:
I can later provide an example how this can be achieved.

Reply:
An example would be very helpful, thanks.

Reply:
Will do. I will actually apply the changes directly to this PR if you don't mind because my code is not really reusable.